jwst/assign_wcs/util.py

```diff
@@ -769,7 +769,7 @@ def _create_grism_bbox(input_model, mmag_extract=99.0,
                     pts = np.array([[ymin, xmin], [ymax, xmax]])
                     ll = np.array([0, 0])
                     ur = np.array([input_model.meta.subarray.ysize, input_model.meta.subarray.xsize])
-                    inidx = np.all(np.logical_and(ll <= pts, pts <= ur), axis=1)
+                    inidx = np.all(np.logical_and(ll < pts, pts < (ur - 1)), axis=1)
```

Comment:
In writing this comment, I realized that the new logic requires both two columns and two rows. Does this follow the requirements, or is the width of the cross-dispersion direction only required to be one? If so, this might require a bit more careful editing to include the dispersion direction.